ballerina/tests/resources/expected_results/variable_default_null_value_with_non_null_type.json

```diff
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Variable \"age\" of type \"Int!\" has invalid default value: null. Expected type \"Int\", found null",
```

Comment:
Is this message correct? Shouldn't it be `Expected type \"Int!\", found null.`?

If this is a small fix, we can fix it now, otherwise, we can create an issue for this and merge this since this is not a blocker.

Reply:
Ignore the comment. The error message is correct.